### PR TITLE
Add Logstash to readme

### PR DESCRIPTION
**Pull request by Conky5**
Add a Logstash section to the readme following the format of Kibana and
Elasticsearch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,6 +48,25 @@ https://www.elastic.co/products/kibana.
 - [Kibana Discuss Forums](https://discuss.elastic.co/c/kibana) 
 - [Kibana Documentation](https://www.elastic.co/guide/en/kibana/current/index.html)
 
+## Logstash
+
+**Logstash** is an open source, server-side data processing pipeline that
```
Comment (karenzone) [inline README.md]: Consider removing the "open source" portion of the description.  There is an OSS version of Logstash, but users have to make an extra effort to get it. Licensing will be getting more complicated with some basic licensed plugins, too.

Comment (Conky5) [reply README.md]: @karenzone this description was copied from https://www.elastic.co/products/logstash, should it be updated there as well?

Comment (Conky5) [reply README.md]: I have dropped it from this doc with 4f5c96a8b3ff2503f9d109f5cda1941de3cd824c.

Comment (karenzone) [reply README.md]: @Conky5 Thanks for pointing this out. I will research and take appropriate action if necessary to resolve.

**Pull request merged**